app/js/components/names.jsx

```diff
@@ -121,140 +124,125 @@ class Name extends React.Component {
   render() {
     let givenErrorClass = '';
     let familyErrorClass = '';
-    if(this.state.errorGivenName && this.state.display === 'edit') {
+    if (this.state.errorGivenName && this.state.display === 'edit') {
       givenErrorClass = 'has-error'
     }
-    if(this.state.errorFamilyName && this.state.display === 'edit') {
+    if (this.state.errorFamilyName && this.state.display === 'edit') {
       familyErrorClass = 'has-error'
     }
-    return(
+    return (
       <div>
         <form className='form-horizontal'>
           {this.state.display === 'view' &&
             <div className='form-group'>
               <div className='col-sm-2'>
-                  <input className='btn btn-success form-control'
-                         name='edit'
-                         type='button'
-                         value='Edit Name'
-                         onClick={ this.handleEdit }
-                  />
+                <input className='btn btn-success form-control'
+                  name='edit'
+                  type='button'
+                  value='Edit Name'
+                  onClick={this.handleEdit}
+                />
               </div>
             </div>
           }
 
-          <div className='form-group '>
-              <label className='control-label col-sm-2'> Preferred </label>
-              <div className='col-sm-2'>
-                  <input
-                      name='username'
-                      type='checkbox'
-                      checked={this.state.preferred}
-                      disabled={
-                                  this.state.display === 'view' ?
-                                   'disabled' : null
-                                }
-                  />
-              </div>
-          </div>
-
-          <div className={'form-group ' + givenErrorClass }>
-              <label className='control-label col-sm-2'> Given Name* </label>
-              <div className='col-sm-5'>
-                  <input  className='form-control'
-                          name='givenName'
-                          type='text'
-                          value={ this.state.givenName }
-                          onChange={ this.handleChange }
-                          readOnly={
-                                    this.state.display === 'view' ?
-                                    'readonly' : null
-                                    }
-                  />
+          <div className={'form-group ' + givenErrorClass}>
+            <label className='control-label col-sm-2'> Given Name* </label>
+            <div className='col-sm-5'>
+              <input className='form-control'
+                name='givenName'
+                type='text'
+                value={this.state.givenName}
+                onChange={this.handleChange}
+                readOnly={
+                  this.state.display === 'view' ?
+                    'readonly' : null
+                }
+              />
```

Comment:
Indentation can be improved for the readonly property

Reply:
corrected.